qa/testing_test.go

```diff
@@ -332,3 +332,55 @@ func TestResourceCornerCases(t *testing.T) {
 func TestAssertErrorStartsWith(t *testing.T) {
 	AssertErrorStartsWith(t, fmt.Errorf("abc"), "a")
 }
+
+func TestGetRawConfig(t *testing.T) {
```

Comment:
Can we use `t.Run()` for each of these test cases? Especially panic test cases: you won't be able to add new test cases after the last one as there will be a panic. Start with
```go
type convertToValueTestCase struct {
  input any
  expected cty.Value
}
```